src/index.js

```diff
@@ -12,6 +12,17 @@ function suggestionTemplate (result) {
   return result && '<strong>' + result.name + '</strong>' + path
 }
 
+// If enabled, this function provides the GOV.UK Registers team with information about your hostname in order to track the component's usage
+function initGoogleAnalyticsTracking () {
```

Comment:
Should this accept trackingID as a variable? What if they want to do their own tracking?

Reply:
This tracking logic does not bind to anything in the component, and just tracks global behaviour for any page this component ends up on.

If they wanted to track usage they'd do it however they're already doing it.

Reply:
+1